source/Halibut/HalibutRuntime.cs

```diff
@@ -338,6 +338,16 @@ public void Dispose()
 
         public async ValueTask DisposeAsync()
         {
+            foreach (var pendingRequestQueue in this.queues.Values)
+            {
+                try
```

Comment:
Don't we have an extension method somewhere that tries to dispose? Or is that Tentacle? 🤔 

Not a big deal, but it would be nice to use it if we have it (for consistency)

Reply:
I could not see one.